docs/migration.md

```diff
@@ -71,6 +75,55 @@ from pydantic.v1.utils import lenient_isinstance
 
 Pydantic V1 documentation is available at [https://docs.pydantic.dev/1.10/](https://docs.pydantic.dev/1.10/).
 
+
+#### Using Pydantic v1 features in a v1/v2 environment
+
+As of `pydantic>=1.10.17`, the `pydantic.v1` namespace can be used within V1.
+This makes it easier to migrate to V2, which also supports the `pydantic.v1`
+namespace. In order to unpin a `pydantic<2` dependency and continue using V1
+features, take the following steps:
+
+1. Replace `pydantic<2` with `pydantic>=1.10.17`
+2. Find and replace all occurrences of:
+
+```python test="skip" lint="skip" upgrade="skip"
+from pydantic.<module> import <object>
+```
+
+to:
+
+```python test="skip" lint="skip" upgrade="skip"
+from pydantic.v1.<module> import <object>
+```
+
+Imports for `pydantic` v1 features for different versions of pydantic.
+
+=== "`pydantic<3`"
+
+    ```python test="skip" lint="skip" upgrade="skip"
+    # Try/except usable for multiple versions of pydantic.
```

Comment:
Maybe move these comments outside of the code block, just into the tab for some context?